src/sphinx_book_theme/assets/styles/content/_admonitions.scss

```diff
@@ -1,20 +1,141 @@
+    > .admonition-title {
+      background-color: rgba(var(--pst-color-admonition-note), 0.1);
+      &::after {
+        color: rgba(var(--pst-color-admonition-note), 1);
+      }
+    }
+  }
+  // Admonition hint styling
+  &.hint {
+    border-color: rgba(var(--pst-color-admonition-hint), 1);
+    > .admonition-title {
+      background-color: rgba(var(--pst-color-admonition-hint), 0.1);
+      &:before {
+        background-color: rgba(var(--pst-color-admonition-hint), 0.1);
+      }
+      &:after {
+        color: rgba(var(--pst-color-admonition-hint), 1);
+      }
+    }
+  }
+  // Admonition tip styling
+  &.tip {
+    border-color: rgba(var(--pst-color-admonition-tip), 1);
+    > .admonition-title {
+      background-color: rgba(var(--pst-color-admonition-tip), 0.1);
+      &:before {
+        background-color: rgba(var(--pst-color-admonition-tip), 0.1);
+      }
+      &:after {
+        color: rgba(var(--pst-color-admonition-tip), 1);
+      }
+    }
+  }
+  // Admonition caution styling
+  &.caution {
+    border-color: rgba(var(--pst-color-admonition-caution), 1);
+    > .admonition-title {
+      background-color: rgba(var(--pst-color-admonition-caution), 0.1);
+      &:before {
+        background-color: rgba(var(--pst-color-admonition-caution), 0.1);
+      }
+      &:after {
+        color: rgba(var(--pst-color-admonition-caution), 1);
+      }
+    }
+  }
+  // Admonition attention styling
+  &.attention {
+    border-color: rgba(var(--pst-color-admonition-attention), 1);
+    > .admonition-title {
+      background-color: rgba(var(--pst-color-admonition-attention), 0.1);
+      &:before {
+        background-color: rgba(var(--pst-color-admonition-attention), 0.1);
+      }
+      &:after {
+        color: rgba(var(--pst-color-admonition-attention), 1);
+      }
+    }
+  }
+  // Admonition danger styling
+  &.danger {
+    border-color: rgba(var(--pst-color-admonition-danger), 1);
+    > .admonition-title {
+      background-color: rgba(var(--pst-color-admonition-danger), 0.1);
+      &:before {
+        background-color: rgba(var(--pst-color-admonition-danger), 0.1);
+      }
+      &:after {
+        color: rgba(var(--pst-color-admonition-danger), 1);
+      }
+    }
+  }
+  // Admonition error styling
+  &.error {
+    border-color: rgba(var(--pst-color-admonition-error), 1);
+    > .admonition-title {
+      background-color: rgba(var(--pst-color-admonition-error), 0.1);
+      &:before {
+        background-color: rgba(var(--pst-color-admonition-error), 0.1);
+      }
+      &:after {
+        color: rgba(var(--pst-color-admonition-error), 1);
+      }
+    }
+  }
   // Admonition seealso styling
   &.seealso {
     border-color: #28a745;
-
-    .admonition-title {
+    > .admonition-title {
       background-color: rgba(40, 167, 70, 0.1);
 
-      &:before {
+      &:after {
         color: #28a745;
         content: "\f064";
       }
     }
   }
+  // Admonition warning styling
+  &.warning {
```

Comment:
What happens if we just delete our `_admonitions.scss` file entirely? IMO that should be our goal.

Reply:
Hi @choldgraf, thank you for your input on this. Very valuable point of inheriting as much as we can without paying too much heed to backward compatibility styling. I have removed the `_admonitions.scss` and also some colors defined in 'src/sphinx_book_theme/assets/styles/abstracts/_variables.scss', which we will directly inherit from pst.